File: sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    height = get_height()
    for i in range(height):
        for j in range(height):
            if i + j >= height - 1:
                print("#", end="")
            else:
                print(" ", end="")
        print()


def get_height():
    while True:
        try:
            height = get_int("Height: ")
            if 0 < height <= 8:
                return height
        except ValueError:
            print("Not an integer")


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt for height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Prompt for building
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height; j++)
            if (i + j >= height - 1)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        printf("\n");
    }
}